final-project/index.html

```diff
@@ -5,10 +5,427 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Document</title>
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.4.2/css/all.css" integrity="sha384-/rXc/GQVaYpyDdyxK+ecHPVYJSN9bmVFBvjA/9eOB+pb3F2w2N6fc5qB9Ew5yIns" crossorigin="anonymous">
   <link rel="stylesheet" href="./dist/styles.css">
 </head>
 <body>
-  <h1>Hello world!</h1>
+  <div id="container">
+    <!-- Header Section -->
+    <header class="main-container header">
+      <div class="sub-container header__content">
+        <div class="header__section">
+          <div class="header__logo"></div>
+          <div class="menu">
+            <ul class="menu__items-list">
+              <li><a href="#" class="menu__item--selected">Home</a></li>
+              <li><a href="#" class="menu__item">Blog</a></li>
+              <li><a href="#" class="menu__item">Features</a></li>
+              <li><a href="#" class="menu__item">Pricing</a></li>
+              <li><a href="#" class="menu__item">Documentation</a></li>
+            </ul>
+          </div>
+        </div>
+        <div class="header__section">
+          <div class="social-networks">
+              <a href="#" class="social-networks__item"><i class="fab fa-facebook-f"></i></a>
+              <a href="#" class="social-networks__item"><i class="fab fa-telegram-plane"></i></a>
+              <a href="#" class="social-networks__item"><i class="fab fa-twitter"></i></a>
+              <a href="#" class="social-networks__item"><i class="fab fa-medium-m"></i></a>
+          </div>
+          <a href="#" class="button button--blue">Get Started</a>
+          <a id="menu-mobile-button" href="#">
+            <i class="fas fa-bars fa-2x"></i>
+          </a>
+        </div>
+      </div>
+    </header>
+  
+    <!-- Body -->
+    <div class="body">
```

Comment:
Another opportunity to use semantic HTML, this can be the `<main>` element, and for each part of the page use `<section>`